compiler/packages/babel-plugin-react-compiler/src/CompilerError.ts

```diff
@@ -18,6 +18,11 @@ export enum ErrorSeverity {
    * Code that breaks the rules of React.
    */
   InvalidReact = 'InvalidReact',
+  /**
+   * Code that breaks the rules of React, but which should not be reported in ESLint due to
+   * false positives.
+   */
+  SilentInvalidReact = 'SilentInvalidReact',
```

Comment:
hmm wait, doesn't this PR need to update the ValidateRefAccess pass with the new severity too?

Reply:
yeah i had a bad revert, look now!